lib/replicate.js

```diff
@@ -46,6 +46,8 @@ function replicate (feed, initiator, opts) {
 }
 
 function Peer (feed, opts) {
+  if (opts.extensions) throw new Error('Per peer extensions is deprecated. Use feed.registerExtension instead')
```

Comment:
If it's just deprecated it shouldn't throw an error. Update error message to say "not supported"